sw_parser/templates/sw_parser/log/elemental_rift/summary.html

```diff
@@ -104,4 +104,61 @@
         </table>
     </div>
 
+	<div class="panel panel-default">
+        <div class="panel-heading">
+            <p class="panel-title">Rune Drops</p>
+        </div>
+        <table class="table">
+            <thead>
+            <tr>
+                <th>Grade</th>
+                <th>Logs</th>
+                <th>Avg. Rarity</th>
+                <th>Avg. Stars</th>
+                {% for item in rune_list.values %}
+                <th><img style="background: url({{ img_url_prefix }}runes/bg_{{ item.avg_quality }}.png) no-repeat center; background-size: contain;" src="{{ img_url_prefix }}runes/{{ item.name }}.png" class="monster-inline" />{{ item.name }}</th>
```

Comment:
See comment in views.py - certain cases can lead to item.avg_quality being undefined which results in an img url of `/static/herders/images/runes/bg_.png` and a 404